src/pdn/doc/define_pdn_grid.md

```diff
@@ -15,6 +15,7 @@ For specifying a power grid over macros in the design:
   % define_pdn_grid
     -macro \
     [-name name] \
+    [-grid_over_pins] \
```

Comment:
I think this name is a little too ambiguous, since pins can refer to power/ground and signal pins, maybe it should be `-limit_grid_to_pins` or something that indicates it's going to reduce the grid.

The "sibling" flag to this would be `-no_grid_over_signal_pins` (something like that), so the straps are not added when they overlap with the signal pins (atleast the layer directly above the pins). This would have the effect of generating the routing channels to pins.

Reply:
It looks be me that -grid_over_pins should be the default and we should have something like -cover_whole_macro as the opposite (or just not have an opposite).

Reply:
We could go a bit more wordy with the flag -grid_over_pg_pins, Even if we make this default, there would be no harm having aswitch for it.

Reply:
@arlpetergadfort - to answer the question you raised above: The -grid_over_pins (or -grid_over_pg_pin) has no relation to signal pin on the macro, purely for the power/ground pins. If signal pins are in the center of the macro then there is a potential for them to be blocked by power/ground straps. If you have any specific examples of this, then I'd like to take a look and see if they can be addressed somehow,